problems/3-delegation/problem.md

```diff
@@ -18,20 +21,53 @@ DID => your program => delegation
 
 Then the workshop will be able to **invoke** the capability to list the items _you've_ uploaded to _your_ space. It'll verify it can find that cat gif you uploaded in the previous exercise.
```

Comment:
Since this changed in the previous exercise:

```suggestion
Then the workshop will be able to **invoke** the capability to list the items _you've_ uploaded to _your_ space. It'll verify it can find that cat meme you uploaded in the previous exercise.
```